ingestion/src/metadata/ingestion/source/database/mssql/usage.py

```diff
@@ -37,32 +25,3 @@ class MssqlUsageSource(MssqlQueryParserSource, UsageSource):
         AND lower(t.text) NOT LIKE '%%create%%function%%'
         AND lower(t.text) NOT LIKE '%%declare%%'
         """
```

Comment:
this `get_sqlalchemy_engine_dateformat` would be useful incase of server in different language.
these were recent PRs we did regarding the same.
[lineage](https://github.com/open-metadata/OpenMetadata/pull/17239/files)
[usage](https://github.com/open-metadata/OpenMetadata/pull/17341/files)
removing timezone form datetime object is alright but we need to convert same string in appropriate format.